### PR TITLE
Migration information to migrate google maps to Bing Maps

### PR DESCRIPTION
**Pull request by stevemunk**

### CODE REVIEW

File: BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md [hunk 1]
```diff
@@ -0,0 +1,2197 @@
...
+services: bing-maps
+---
+
+# Migrating Google Maps to Bing Maps
+
+This article provides the information you need to migrate an application built using the Google Maps V3 JavaScript API to a Bing Maps application, built using the Bing Maps V8 web control API. This includes a suggested migration plan, useful tools and support, migration code examples, API class mapping along with other tips to help ensure a successful migration.
+
+**Why Migrate?**
+
+If you are still evaluating Bing Maps to determine if you should migrate, you may find the following information helpful:
```
Comment (eriklindeman) [inline BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md:19]:
```suggestion
If you are evaluating Bing Maps to determine if you should migrate, you may find the following information helpful:
```

File: BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md [hunk 2]
```diff
@@ -0,0 +1,2197 @@
...
+   Docs](https://docs.microsoft.com/bingmaps/getting-started/bing-maps-dev-center-help/creating-a-bing-maps-account).
+
+1. Create a Bing Maps key.
+
+    a.  We invite you to use the "basic" key type and free use terms
+        outlined in the [Bing Maps Terms of Use] (https://www.microsoft.com/maps/product)
+        (TOU) as you migrate to Bing Maps. Production applications outside the free
+        use thresholds in the TOU will need a Bing Maps Enterprise
+        account. For assistance with Bing Maps licensing, please contact
+        <maplic@microsoft.com> (North/South America) or
```
Comment (eriklindeman) [inline BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md:39]:
```suggestion
        <maplic@microsoft.com> (the Americas) or
```

File: BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md [hunk 3]
```diff
@@ -0,0 +1,2197 @@
...
+        <mapemea@microsoft.com> (rest of world).
```
Comment (eriklindeman) [inline BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md:40]:
```suggestion
        <mapemea@microsoft.com> (Europe, Middle East, Africa, Asia and Australia).
```

File: BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md [hunk 4]
```diff
@@ -0,0 +1,2197 @@
...
+Support team. The Bing Maps developer forums are regularly monitored by community
+developers and by members of the Bing Maps team. You can find and ask questions
+in [Microsoft Q&A](https://docs.microsoft.com/answers/products/).
+
+#### Licensing Queries
+
+If you have licensing-related questions you should take them to your
+Bing Maps account manager if you know who they are. If not, you can send
+queries to the Bing Maps licensing team, and they will assist. For
+queries inside of North or South America, you can contact them at
```
Comment (eriklindeman) [inline BingMaps/getting-started/google-maps-to-bing-maps-migration-guide/Google-Maps-to-Bing-Maps-Migration-Guide.md:101]:
```suggestion
queries inside the Americas, you can contact them at
```

**Review by IoTGirl — COMMENTED** [review]
I am passing review and approval to Clemens

**Review by cschotte — COMMENTED** [review]
All good

**Review by cschotte — APPROVED** [review]
All good

**Pull request merged**